step/step.go

```diff
@@ -693,7 +693,7 @@ func (s XcodebuildArchiver) createCodesignManager(config Config) (codesign.Manag
 
 	codesignConfig, err := codesign.ParseConfig(codesignInputs, s.cmdFactory)
 	if err != nil {
-		return codesign.Manager{}, fmt.Errorf("issue with input: %s", err)
+		return codesign.Manager{}, err
```

Comment:
What's the reason for removing the extra context here?

Reply:
With the top-level error message (`Failed to process Step inputs`), we already mention that the issue is around the Step Inputs.

From this:
```
Failed to process Step inputs:
  failed to prepare automatic code signing:
    issue with input: code signing certificate URL is not specified
```

to this:
```
Failed to process Step inputs:
  failed to prepare automatic code signing:
    code signing certificate URL is not specified
```